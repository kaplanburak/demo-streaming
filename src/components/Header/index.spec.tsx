import React from "react";
import { shallow } from "enzyme";
import { Header } from "./index";

describe("Header component", () => {
  it("should render with page title", () => {
    const header = shallow(<Header pageTitle={"Test Title"} />);
    expect(header).toMatchSnapshot();
  });
});
