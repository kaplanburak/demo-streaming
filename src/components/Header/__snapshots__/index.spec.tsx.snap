// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render with page title 1`] = `ShallowWrapper {}`;
